app/controllers/webview/drug_stocks_controller.rb

```diff
@@ -21,7 +21,7 @@ def create
     DrugStocksCreator.call(user: current_user,
                            facility: @current_facility,
                            for_end_of_month: @for_end_of_month,
-                           drug_stocks_params: safe_params[:drug_stocks])
+                           drug_stocks_params: drug_stocks_params)
```

Comment:
We went from passing `safe_params[:drug_stocks]` to `safe_params[:drug_stocks].values`, but I don't see any corresponding change in the behavior of the DrugStocksCreator. Is this intentional?

Reply:
We've changed the form's structure in the view to let us split the form fields HTML without any JS. 

Earlier it was an array of protocol drug objects
```
[{protocol_drug_id: , in_stock:, received: }, ....]
```

Now it is a hash that looks like
```
{ 
  "0" => {protocol_drug_id: , in_stock:, received: }, 
  "1" => {protocol_drug_id: , in_stock:, received: }
   ...
}
```

Reply:
Ah understood 👍